tasks/mutations/task_template_mutations.ts

```diff
@@ -58,14 +58,19 @@ export const useWardTaskTemplateQuery = (wardId? : string, onSuccess: (data: Tas
   })
 }
 
-export const usePersonalTaskTemplateQuery = (createdBy? : string, onSuccess: (data: TaskTemplateDTO[]) => void = noop) => {
+export const useAllTaskTemplatesByCreator = (
+  createdBy?: string,
+  onSuccess: (data: TaskTemplateDTO[]) => void = noop,
+  onlyPrivate: boolean = false
+) => {
```

Comment:
Consider changing it to `useAllTaskTemplatesByCreator({ createdBy: ..., onlyPrivate: ..., onSuccess: ... })`. Having a boolean property for which you only know the name (and thus impact) when hovering over the parameter in your editor seems like a bad idea in this case